src/AppInstallerCLICore/Workflows/ArchiveFlow.h

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+#include "ExecutionContext.h"
+
+namespace AppInstaller::CLI::Workflow
+{
+    void ExtractInstallerFromArchive(Execution::Context& context);
```

Comment:
See other workflow tasks for the comments to place above these.

Reply:
Added comments to workflow task